Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -166,41 +98,17 @@ - (void)dayPicker:(MZDayPicker *)dayPicker didSelectDay:(MZDay *)day
     }
 }
 
+
```

Comment:
For clarification, what do you mean by remove errant newline? The pattern I used was one newline between methods, and then two newlines between the end of the last method under one pragma mark and then the next pragma mark, just for spacing differentiation. Do you just want one newline between sections as well?